vllm/executor/ray_xpu_executor.py

```diff
@@ -82,7 +84,13 @@ def __init__(
         self.extra_execute_model_run_workers_kwargs: Dict[str, Any] = {}
 
     def _init_executor(self) -> None:
-        pass
+        if USE_RAY_COMPILED_DAG:
```

Comment:
have you tested with xpu backend (or is it tested in CI)? I think if not, we just should not change this file imo (and not support spmd worker for tpu first).